docs/Routes.md

```diff
@@ -60,6 +60,32 @@ fastify.route({
 })
 ```
 
+### Declaring an array of routes
```

Comment:
It may be better to avoid a dedicated section. Maybe a little reminder is more appropriate


```js
const routes = [
  {
    method: 'GET',
    path: '/',
    handler: myHandler
  },
  ... // other route definition
]

routes.forEach(r => fastify.route(r))
```

Reply:
@allevo I think this is a good idea, I just uploaded it, could it validate?